portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/** fetches harded coded string and display them in the broswer page onload*/
+async function displayGreetins(){
+const responseFromserver=await fetch("/hello");
+const textFromResponse=await responseFromserver.text();
+
+const  greetingContainer=documnet.getElementById('greetingCon');
```

Comment:
Is this code working/needed? This function isn't being called anywhere and the server code at /hello is called by clicking on that button "see greeting." 

As a side note, there's a minor typo so it should be:
`const greetingContainer = document.getElementById('greetingCon');`
But also in the current files, there's nothing with the ID `greetingCon`.

If this function isn't being used anywhere, you can just delete it.

Reply:
Yea, it should have been deleted..I didnt call the code